rclcpp_lifecycle/src/lifecycle_node.cpp

```diff
@@ -191,7 +162,6 @@ LifecycleNode::~LifecycleNode()
   node_timers_.reset();
   node_logging_.reset();
   node_graph_.reset();
-  node_base_.reset();
```

Comment:
this is actually bug fix, i will re-submit this cosmetic after revert as follow-up.